context/app/static/js/stores/useSearchViewStore.js

```diff
@@ -1,7 +1,9 @@
 import create from 'zustand';
 
 const useSearchViewStore = create((set) => ({
-  searchView: new URLSearchParams(window.location.search).get('view') || 'table',
+  searchView:
+    (window.location.pathname.startsWith('/search') && new URLSearchParams(window.location.search).get('view')) ||
+    'table',
```

Comment:
I'm not sure this is needed: `new URLSearchParams(window.location.search).get('view') || 'table'` will evaluate to `'table'` on the /dev-search.

Reply:
Unless a user enters `&view=tile` in the url. Which we might not care about?

Reply:
Per our Zoom conversation, we shouldn't worry about users modifying the url and things breaking. Prefer to keep code simpler.